scripts/validate/validate.go

```diff
@@ -0,0 +1,82 @@
+package main
```

Comment:
should we combine all these into a single script?

Reply:
I thought about it. But we want to run those independently of 'generate script'. We use generate script for website docs, artifact hub, etc. (which runs independently). So the idea is to run the validation script separately once we generate all the docs. Also, this script can extend to validate other aspects of the repo and can run as an independent validator.